services/kommander/0.1.0/defaults/cm.yaml

```
@@ -18,6 +18,7 @@ data:
           name: ${certificatesCAIssuerName}
       issuer:
         name: ${certificatesIssuerName}
+        kind: ${certificateIssuerKind}
```

Should these have defaults? e.g.
```suggestion
        kind: ${certificateIssuerKind:-Issuer}
```